web-app/server/compileIR.js

```diff
@@ -0,0 +1,79 @@
+var fs = require('fs');
+var exec = require('child_process').exec;
+var LLFI_BUILD_ROOT = require('./utils/config').LLFI_BUILD_ROOT;
+var execPromise = require('./utils/execPromise').execPromise;
+
+exports.processCompileIR = function (req, res) {
```

Comment:
@syltaxue If not too difficult, could you rename this file and all the references of compileIR to compileToIR? It may be a seemingly subtle detail but this renaming would make it consistent with the other processes.  We are really compiling source code to IR instead of compiling IR code itself.